spec/services/proofing/resolution/result_adjudicator_spec.rb

```diff
@@ -57,98 +57,8 @@
-          result = subject.adjudicated_result
-
-          expect(result.success?).to eq(true)
-        end
-      end
-      context 'LexisNexis fails with attributes covered by AAMVA response' do
-        let(:resolution_success) { false }
-        let(:can_pass_with_additional_verification) { true }
-        let(:attributes_requiring_additional_verification) { [:dob] }
-        let(:state_id_verified_attributes) { [:dob, :address] }
-
-        it 'returns a successful response' do
-          result = subject.adjudicated_result
-
-          expect(result.success?).to eq(true)
-        end
-      end
-
-      context 'LexisNexis fails with attributes not covered by AAMVA response' do
-        let(:resolution_success) { false }
-        let(:can_pass_with_additional_verification) { true }
-        let(:attributes_requiring_additional_verification) { [:address] }
-        let(:state_id_verified_attributes) { [:dob] }
-
-        it 'returns a failed response' do
-          result = subject.adjudicated_result
-
-          expect(result.success?).to eq(false)
-        end
-      end
-
-      context 'LexisNexis fails and AAMVA state is unsupported' do
-        let(:should_proof_state_id) { false }
-        let(:resolution_success) { false }
-
-        it 'returns a failed response' do
-          result = subject.adjudicated_result
-
-          expect(result.success?).to eq(false)
-        end
-      end
-
-      context 'LexisNexis passes and AAMVA fails' do
-        let(:resolution_success) { true }
-        let(:state_id_success) { false }
-
-        it 'returns a failed response' do
-          result = subject.adjudicated_result
-
-          expect(result.success?).to eq(false)
-        end
-      end
-
-      context 'Device profiling fails and everything else passes' do
-        let(:device_profiling_success) { false }
-        let(:device_profiling_review_status) { 'fail' }
-
-        it 'returns a successful response including the review status' do
-          result = subject.adjudicated_result
-
-          expect(result.success?).to eq(true)
-
-          threatmetrix_context = result.extra[:context][:stages][:threatmetrix]
-          expect(threatmetrix_context[:success]).to eq(false)
-          expect(threatmetrix_context[:review_status]).to eq('fail')
-        end
-      end
-
-      context 'Device profiling experiences an exception' do
-        let(:device_profiling_success) { false }
-        let(:device_profiling_exception) { 'this is a test value' }
-
-        it 'returns a failed response' do
-          result = subject.adjudicated_result
-
-          expect(result.success?).to eq(false)
-
-          threatmetrix_context = result.extra[:context][:stages][:threatmetrix]
-          expect(threatmetrix_context[:success]).to eq(false)
-          expect(threatmetrix_context[:exception]).to eq('this is a test value')
-        end
-      end
-    end
-
     context 'double address verification is enabled' do
```

Comment:
Would love to remove this context and un-indent everything inside if that's not too hard 😬

Reply:
that makes sense!